homeassistant/components/plugwise/number.py

```diff
@@ -47,12 +48,14 @@ class PlugwiseNumberEntityDescription(
         command=lambda api, number, value: api.set_number_setpoint(number, value),
         device_class=NumberDeviceClass.TEMPERATURE,
         entity_category=EntityCategory.CONFIG,
-        native_max_value_key="upper_bound",
-        native_min_value_key="lower_bound",
-        native_step_key="resolution",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
-        native_value_key="setpoint",
+        native_max_value_fn=lambda data: data["upper_bound"],
+        native_min_value_fn=lambda data: data["lower_bound"],
+        native_step_key_fn=lambda data: data["resolution"],
+        native_value_fn=lambda data: data["setpoint"],
+        number_fn=lambda data: data["maximum_boiler_temperature"],
```

Comment:
The naming here feels weird.
Maybe it should be called `actuator_fn`?
```suggestion
        actuator_fn=lambda data: data["maximum_boiler_temperature"],
```

Reply:
Sure, no problem, will implement the change soon.

Reply:
Name-change implemented.